Test/comp/globalNamespace/globalNamespaceRunConfig.toml

```diff
@@ -0,0 +1,5 @@
+includes=["./globalNamespaceRun.dfy"]
+
+[options]
+outer-namespace=["my","happy"]
```

Comment:
Apologies, I accidentally some important words in that sentence :). I meant "Also I'm assuming we'll need a way to specify options that are only provided when compiling to a given backend <**in project files**>"

> If you use a project file, then all options in the project file that apply to the command you're using are applied, so you can have

That's a bit dicey because it assumes the backend specific options are all uniquely named, but I bet more than one backend will end up naturally providing `--outer-module`. It also won't hold up as a general method for providing backend-specific options for the same reason.

As I said though I think it's fine to leave this out of scope for this PR - users can still use project files for common options but provide these backend specific ones when invoking the CLI, such as `dafny translate cs dfyconfig.toml --outer-namespace My.Happy`